src/rcc.rs

```diff
@@ -143,6 +151,33 @@ impl CFGR {
         self
     }
 
+    /// Enables write access to the registers in the backup domain
+    pub fn access_backup_domain(&mut self, pwr: &mut PWR) -> BackupDomain {
```

Comment:
Maybe backup_domain is enough. Later, we can implement access to backup registers on this struct.